src/pages/api/threads/post/[id].ts

```diff
@@ -39,7 +41,7 @@ export default async function handler(
     console.error(error);
     res
       .status(500)
-      .json({ isSuccess: false, message: 'Internal Server Error' });
+      .json({ isSuccess: false, message: error.message || 'Internal Server Error' });
```

Comment:
The issue states that, "Let users know they can't convert images without caption." If I will remove this it will still show "Internal Server Error" on client.

Reply:
i know, but it doesn't concern if you want that changed, create a new issue

Reply:
Sure I've added the necessary changes.